### DISCUSSION TITLE
Is FunctionCallContent stable enough to use?

### DISCUSSION THREAD

**Discussion by noeltupas** [microsoft/semantic-kernel#7278] [Q&A] [answered]
My original inquiry here:

https://github.com/microsoft/semantic-kernel/discussions/6589

I'd like to know if FunctionCallContent is stable enough to use since we want to take telemetry on functions uses during a prompt flow. This class would make that much easier. However it's marked as Experimental:

![image](https://github.com/user-attachments/assets/5609d3cf-e61d-40c3-8129-3aa33c12bc9c)

If someone could please let me know what the long term intentions are with this type I'd appreciate it.

**Comment by noeltupas** [comment]
[moonbox3](https://github.com/moonbox3) I moved my question here to address it outside the context of an already answered question~

**Comment by sophialagerkranspandey** [comment, marked as answer]
Hi @noeltupas , yes the Function CallContent is stable enough to use.

**Reply by noeltupas** [reply]
Ok thanks~ I'll take that as an indicator of long-term availability. 🫡😉
